test/recipes/30-test_evp.t

```diff
@@ -187,7 +187,7 @@ sub test_errors { # actually tests diagnostics of OSSL_STORE
 }
 
 SKIP: {
-    skip "DSA not disabled", 2 if !disabled("dsa");
+    skip "DSA not disabled", 2 if !disabled("dsa") || disabled("err");
```

Comment:
So if both DSA and ERR are disabled, we get the message "DSA not disabled" - won't that be confusing?

Reply:
Good point. I updated the error message.